### ISSUE TITLE
Make GWT nature addition optional for Projects that refer to GWT maven plug-ins in their pom

### ISSUE THREAD

**Issue by dfht** [gwt-plugins/gwt-eclipse-plugin#344]
Some of my GWT projects consist entirely of generated artefacts (from models etc.).
To build them out I package them as gwt-lib (Thomas Broyer's Maven GWT plug-in).

I see that the MavenProjectConfigurator always adds the GWT nature to such projects. 

For my projects that have only generated resources in, I do not want the GWT nature to be added as it results in extra builders, validation, etc. that aren't necessary (as I already know that the artefacts are valid).

For some projects (hand maintained code) I do want the natures. Could you please add a project by project flag so I can avoid the nature addition in some cases?

I ask because we have a very large number of projects, and the addition of GWT natures seems to lead to a lot of extra builder invocation that in some cases we could avoid in order to boost the performance of our IDE.

**Comment by dfht** [comment]
Alternatively, separating the maven configurator into a separate (optional) feature would of course achieve the same goal as I can then choose which projects I want the nature on rather than it just getting added by M2E updates etc.

**Comment by branflake2267** [comment]
Good idea. I've added it to my todos. I'm working on the plugin this week.
I think it would be nice to turn it off.
